sdk/python/tests/compiler/testdata/input_artifact_raw_value.py

```diff
@@ -14,18 +14,28 @@
 
 import sys
 
-from kfp import dsl
+from kfp import dsl, components
 from pathlib import Path
 
 sys.path.insert(0, __file__ + '/../../../../')
 
+COMP_STR = """
+name: component-with-inline-input-artifact
+description: A simple component
+inputs:
+  - {name: text, type: String}
+implementation:
+  container:
+    image: alpine
+    command:
+    - cat
+    - {inputValue: text}
+"""
+comp_op = components.load_component_from_text(COMP_STR)
+
 
 def component_with_inline_input_artifact(text: str):
-    return dsl.ContainerOp(
-        name='component_with_inline_input_artifact',
-        image='alpine',
-        command=['cat', dsl.InputArgumentPath(text, path='/tmp/inputs/text/data', input='text')],  # path and input are optional
```

Comment:
In this example, we want to test the `dsl.InputArgumentPath` DSL, so we can't simply just put text as argument.

Reply:
then need to skip this file and update it when we figure out how to use component.xml with dsl.InputArgumentPath

Reply:
reverted it and I have a comment at issue [670](https://github.com/kubeflow/kfp-tekton/issues/670#issuecomment-895371381)